db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/DataDictionaryImpl.java

```diff
@@ -7134,7 +7135,7 @@ protected ConglomerateDescriptor bootstrapOneIndex(SchemaDescriptor sd,
                 null, //default sort order
                 null, //default collation id's for collumns in all system congloms
                 indexProperties, // default properties
-                TransactionController.IS_DEFAULT); // not temporary
+                TransactionController.IS_DEFAULT, 100); // not temporary
```

Comment:
use constant enum instead.
Also, the //not temporary comment refers to TransactionController.IS_DEFAULT, which becomes less apparent after your change. I suggest bringing the priority to the next line.